contrib/localnet/solana/start-solana.sh

```diff
@@ -6,7 +6,7 @@ solana-keygen new -o /root/.config/solana/id.json --no-bip39-passphrase
 
 solana config set --url localhost
 echo "starting solana test validator..."
-solana-test-validator &
+solana-test-validator --limit-ledger-size 50000000 &
```

Comment:
Imo this value can be lowered if needed `50000000` , `1000000` should be enough even for the upgrade tests

Reply:
I think this is the minimum default value on mainnet. I was thinking it might be nice to have similar behavior.

Reply:
okay, that works, primary reason I mentioned the smaller number is to try and make the solana container a bit lighter.

Reply:
Also it only seems to prune on restart so this is really only relevant to users utilizing the persistence feature.

You can totally just reset the persistence state whenever you want to as well.